CONTRIBUTING.md

```diff
@@ -27,3 +27,27 @@ your patch's fault. Help to fix the root cause if something else has
 broken the CI, before pushing.
 
 *Happy Hacking!*
+
+# How to run a CI pipeline on Pull Requests from external contributors
+_An instruction for maintainers_
+
+## TL;DR:
+- Review the PR
+- If and only if it looks **safe** (i.e. it doesn't contain any malicious code which could expose secrets or harm the CI), then:
+    - Press the "Approve and run" button in GitHub UI
+    - Add the `approved-for-ci-run` label to the PR
+Repeat all steps after any change to the PR.
```

Comment:
```suggestion

Repeat all steps after any change to the PR.
```

This is rendered on the same line as the thing before. I think you can resolve this by adding a newline.